internal/resource/default_user_secret.go

```diff
@@ -87,6 +94,61 @@ func (builder *DefaultUserSecretBuilder) Update(object client.Object) error {
 	return nil
 }
 
+func (builder *DefaultUserSecretBuilder) addPortsToSecret() {
+	const (
+		AMQPPort  = "5672"
+		AMQPSPort = "5671"
+	)
+	portNames := map[v1beta1.Plugin]string{
+		"rabbitmq_mqtt":      "mqtt-port",
+		"rabbitmq_stomp":     "stomp-port",
+		"rabbitmq_stream":    "stream-port",
+		"rabbitmq_web_mqtt":  "web-mqtt-port",
+		"rabbitmq_web_stomp": "web-stomp-port",
```

Comment:
From an application perspective, does it ever need the management port?

Reply:
I don't think management should be needed